indexer/src/main/java/main/Indexer.java

```diff
@@ -0,0 +1,87 @@
+package main;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.inject.Guice;
+import com.google.inject.Inject;
+import juice.modules.IndexerModule;
+import juice.modules.StrictExplicitBindingModule;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.elasticsearch.action.index.IndexRequest;
+import org.elasticsearch.client.RequestOptions;
+import org.elasticsearch.client.RestHighLevelClient;
+
+import javax.inject.Singleton;
+import java.time.Duration;
+import java.util.Collections;
+import java.util.Map;
+
+@Singleton
+public class Indexer {
+
+        private final KafkaConsumer<String, String> consumer;
+        private final RestHighLevelClient client;
```

Comment:
For a reader, it is hard to understand this is an Elasticsearch client. Let's rename the variable

Reply:
done